docs/source/credentials.rst

```diff
@@ -0,0 +1,85 @@
+=============================
+Credentials for smaht-submitr
+=============================
+
+If you have been designated as a submitter for the project,
+and plan to use spreadsheet-based submission system (``smaht-submitr``)
+to view, upload, create, or update data from/to the SMaHT data portal,
+you are required to obtain and setup an access and secret key from the SMaHT data portal.
+These must stored in a specific format and file on your local system.
+Please follow these steps below to get your access keys and configure your local system for ``smath-submitr`` usage.
+
+|
+
+**Obtaining SMaHT Data Portal Access Keys**
+
+#. Log in to the SMaHT `website <https://data.smaht.org>`_ with your username (email) and password. If you have not yet created an account, see `this page </docs/user-guide/account-creation>`_ for instructions.
+#. Once logged in, go to your **Profile** page by clicking **Account** on the upper right corner of the page.
+#. On your profile page, click the green **Add Access Key** button, and copy the **Access Key ID** and **Secret Access Key** values from the popup page. Note that *once the pop-up page disappears you will not be able to see the secret access key value*. However, if you forget/lose your secret key you can always delete and add new access keys from your profile page at any time.
+#. Store these values the file ``~/.smaht-keys.json`` on your local machine; see the next section for details.
+
+|
+
+**Storing SMaHT Data Portal Access Keys**
+
+Once you've obtained access and secret keys (per the previous) section, they should be stored in a file on your local machine called ``~/.smaht-keys.json``. (Note that the ``~`` there refers to your local home directory). The format of this file should look like this::
+
+   {
+       "data": {
+           "key": "<your-access-key-id>",
+           "secret": "<your-secret-access-key>",
+           "server": "https://data.smaht.org"
+       },
+       "staging": {
+           "key": "<your-access-key-id>",
+           "secret": "<your-secret-access-key>",
+           "server": "https://staging.smaht.org"
+       }
```

Comment:
Especially here I feel like it is not good to include staging since this is outward facing